common/Makefile.am

```diff
@@ -30,6 +30,7 @@ libyami_common_ldflags = \
 	$(LIBVA_LIBS) \
 	$(LIBVA_DRM_LIBS) \
 	-lpthread \
+	-std=c++11 \
```

Comment:
Okay, @uartie.
But as I see it,  std::strtol()  requires c++11. 
If so, I will replace std::strtol with atol().